opl/pass_or_fail.py

```diff
@@ -200,8 +206,17 @@ def main():
     if not args.dry_run:
         for d_type in args.decisions_type:
             if d_type == "elasticsearch":
+                if hasattr(args, "es_server_verify"):
+                    # disable SSL verification
+                    opl.http.insecure()
                 opl.investigator.elasticsearch_decisions.store(
-                    args.decisions_es_server, args.decisions_es_index, info_all
+                    args.decisions_es_server,
+                    args.decisions_es_index,
+                    info_all,
+                    es_server_user=getattr(args, "decisions_es_server_user", None),
+                    decisions_es_server_pass_env_var=getattr(
```

Comment:
Tihs is not a requirement, but to align with `es_server_user` I would rename `decisions_es_server_pass_env_var` to `es_server_pass_env_var`. Or rename `es_server_user` to `decisions_es_server_user`